### PR TITLE
in start_bundle_coupled fail on missing files/dirs

### PR DESCRIPTION
**Pull request by orichters**
## Purpose of this PR

- If you somehow misspecified the files / folders, `start_bundle_coupled` broke quite late with unclear message. Rather fail early.
- in `output.R`, if no folder contains gdx files (and mif if needed), fail and tell the user instead of printing this nonsense:

```
Please choose runs to be used for output generation:

1,a: all
2,p: Search pattern by regular expression...
3,f: Search by fixed pattern...
```

## Type of change

- [x] Bug fix 

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] All automated model tests pass (`FAIL 0` in the output of `make test-coupled-slurm`)

### CODE REVIEW

File: start_bundle_coupled.R [hunk 1]
```diff
@@ -178,6 +168,21 @@ deletedFolders <- 0
...
+message("path_magpie_oldruns:   ", if (dir.exists(path_magpie_oldruns)) green else red, path_magpie_oldruns, NC)
+message("prefix_runname:        ", prefix_runname)
+message("n600_iterations:       ", n600_iterations)
+message("run_compareScenarios:  ", run_compareScenarios)
+
+if (any(! file.exists(c(path_settings_coupled, path_settings_remind))) ||
+    any(! dir.exists(c(path_remind, path_magpie, path_remind_oldruns, path_magpie_oldruns)))) {
+  stop("Missing files or directories, see in red above.")
+}
+
```
Comment (dklein-pik) [inline start_bundle_coupled.R:185]: Why did you move it downwards?

Comment (orichters) [reply start_bundle_coupled.R:185]: - It needed to be below the `dir.create("output")` to make sure `path_remind_oldruns` exists.
- I first wanted to do it via `errorsfound` and fail at the end of the file, but then I thought it is better to fail immediately. So yeah, one could move it back up some lines, but I think it does not matter.

Comment (dklein-pik) [reply start_bundle_coupled.R:185]: Ok, now one medal for me. It was kind of obvious that 'output' is part of ' path_remind_oldruns' .

**Review by dklein-pik — APPROVED** [review]

File: start_bundle_coupled.R (hunk 1 above)
Comment (dklein-pik) [reply start_bundle_coupled.R:185]: Sorry, I know it is not important and we can leave it as is, but I simply can't shut up: if `output` has just been created the line above there will be nothing to be found in `path_remind_oldruns` anyway, is it?

Comment (orichters) [reply start_bundle_coupled.R:185]: Yes, so no run will be found which is ok, the script does not fail in this case.
